nchp/templates/nginx.conf

```diff
@@ -114,6 +114,11 @@ http {
         {% endif %}
 
         client_max_body_size {{ client_max_body_size }};
+        
+        location = /healthz {
+          return 200;
+          #access_log off;
```

Comment:
Good question... Old habits die hard as far as leaving in comment debug stuff.
I guess I don't see any reason to leave it in.